src/signals/settings/users/containers/Overview/index.js

```diff
@@ -49,9 +49,11 @@ const filtersReducer = (state, action) => {
 
 export const UsersOverviewContainer = ({ pageSize, userCan }) => {
   const history = useHistory();
+  const location = useLocation();
+  const filtersInitialState = location.state && location.state.filters || {};
```

Comment:
Wouldn't `location.state.filters || {};` be enough? As far as I can see, the `location` object always has a `state` prop.

Reply:
Tried that before, but apparently it's not always there...